### PR TITLE
link exercises in tutorials

### PR DESCRIPTION
**Pull request by Aquilaafuadajo**
Hello @sabine @SaySayo @cuihtlauac, I attempted #1780  Please help review and let me know if I'm on the right track 😊 🙏🏽 
I'm still reading through the ocaml tutorials and matching them with the relevant exercises I find

### CODE REVIEW

**Comment by SaySayo** [comment]
This looks good to me! You are on the right track.

**Comment by Aquilaafuadajo** [comment]
okay, Thank you so much @SaySayo  🙏🏽

**Comment by Aquilaafuadajo** [comment]
@SaySayo @sabine @cuihtlauac please help review. Sorry this took a while 🙏🏽, I had to take some time to understand the tutorials in order to link them to reasonable exercises

**Review by christinerose — APPROVED** [review]

**Comment by Dev-JoyA** [comment]
hello @SaySayo have this issue been resolved or can i go ahead and work on it.

**Comment by SaySayo** [comment]
@Aquilaafuadajo is currently working on it. @Dev-JoyA you can go through the list of issues to find another issue you'd like to try.

**Comment by Aquilaafuadajo** [comment]
Hi @SaySayo, I have completed this task, can you please help review 🙏🏽

File: data/exercises/003_nth_element.md
Comment (cuihtlauac) [inline data/exercises/003_nth_element.md:1]: This whole exercise must be reworked.
- The task description is not clear
  - Is about implementing or using `List.nth`?
  - Should it return an option or throw an exception?

File: data/exercises/060_height_balanced_btree_2.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ slug: "60"
 difficulty: intermediate
 tags: [ "binary-tree" ]
 description: "Calculate the maximum and minimum number of nodes, height, and generate all height-balanced binary trees."
+tutorials: [ "labels" ]
```
Comment (cuihtlauac) [inline data/exercises/060_height_balanced_btree_2.md]:
```suggestion
tutorials: [ "basic-data-types" ]
```

File: data/exercises/096_syntax_checker.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ slug: "96"
 difficulty: intermediate
 tags: []
 description: "Check whether a given string conforms to the syntax rules of a legal identifier in a programming language."
+tutorials: [ "mutability-imperative-control-flow" ]
```
Comment (cuihtlauac) [inline data/exercises/096_syntax_checker.md]:
```suggestion
tutorials: [ "basic-data-types" ]
```

**Review by cuihtlauac — DISMISSED** [review]

**Review by sabine — APPROVED** [review]
Thanks @Aquilaafuadajo! :sparkles: Merging this now. :heart:

**Pull request merged**